esp/esp/themes/theme_data/fruitsalad/templates/main.html

```diff
@@ -107,10 +107,23 @@
 	</form>
       </div>
       <div id="contact_info">
-	<span class="title accentcolor">{{ theme.full_group_name }}</span>
-      <span class="normal">{{ theme.contact_info|linebreaksbr }}</span>
-      <span class="normal">E-mail: <a href="mailto:{{ settings.DEFAULT_EMAIL_ADDRESSES.default }}">{{ settings.DEFAULT_EMAIL_ADDRESSES.default }}</a></span>
-      <span class="normal"><a href="/faq.html">FAQ</a> | <a href="/contact.html">contact us</a>{% if theme.facebook_link %} | <a href="{{ theme.facebook_link }}">facebook</a>{% endif %}</span>
+        <span class="title accentcolor">{{ theme.full_group_name }}</span>
+        <span class="normal">{{ theme.contact_info|linebreaksbr }}</span>
+        <span class="normal">E-mail: <a href="mailto:{{ settings.DEFAULT_EMAIL_ADDRESSES.default }}">{{ settings.DEFAULT_EMAIL_ADDRESSES.default }}</a></span>
+        <span class="normal">
+          {% if theme.faq_link == None %}
```

Comment:
This doesn't do what you meant it to do: variables that don't exist default to empty string, not None. Conveniently, themes handles defaults for you (https://github.com/learning-unlimited/ESP-Website/blob/main/esp/esp/themes/controllers.py#L95), so you can safely delete this case.